### PR TITLE
Tests and storybooks

### PR DESCRIPTION
**Pull request by HectorZR**

### CODE REVIEW

File: .storybook/manager.js [hunk 1]
```diff
@@ -2,5 +2,5 @@ import { addons } from '@storybook/addons';
 import { themes } from '@storybook/theming';
 
 addons.setConfig({
-    theme: themes.dark,
-});
\ No newline at end of file
+  theme: themes.normal,
```
Comment (angel-afonso) [inline .storybook/manager.js:5]: That's suspicious ( ? )

Comment (HectorZR) [reply .storybook/manager.js:5]: Hahahahahahaha just to be able to watch fullscreen loader in storybook

File: src/components/textInput/TextInput.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <TextInput name="Example" value="" onChange={() => {}} label="Example" />
+  </Provider>
+);
+
+export default {
+  title: 'TextInput',
+  component: TextInput,
+};
+
+export const NormalTexInput = () => <TextInputWithProvider />;
```
Comment (angel-afonso) [inline src/components/textInput/TextInput.stories.jsx:17]: Typo here, missing letter T in NormalTextInput

Comment (HectorZR) [reply src/components/textInput/TextInput.stories.jsx]: Done

File: src/components/textInput/TextInput.stories.jsx [hunk 3]
```diff
@@ -14,4 +14,4 @@ export default {
   component: TextInput,
 };
 
-export const NormalTexInput = () => <TextInputWithProvider />;
+export const NormalTexTInput = () => <TextInputWithProvider />;
```
Comment (angel-afonso) [inline src/components/textInput/TextInput.stories.jsx:17]: Thank you xD

**Pull request merged**